starter-project-web-vue2/components/yohans/BlogItem.vue

```diff
@@ -0,0 +1,33 @@
+<template>
```

Comment:
It would be better if you guys are creative with the designs. This is almost similar to Abraham's

Reply:
All right cool!